Dockerfile

```diff
@@ -14,7 +14,8 @@ COPY main.go main.go
 COPY api/ api/
 COPY controllers/ controllers/
 COPY pkg/ pkg/
-COPY bindata/ bindata/
+COPY bindata/deployment/ bindata/deployment/
```

Comment:
I don't see we are using this files in the build container so you can remove this and in the release build lets just use COPY without --from

Reply:
Hmm we do this is the metallb.yaml to deploy metallb speaker and controller ?

Reply:
In general, I just don't see the need to copy this yamls into the build container(golang one) and then copy it to the release container you can just copy it directly

Reply:
u need to copy them to the build container so you can build the manager binary ? anyway we can clean this up once the PR is merged and ofcourse any cleanup is welcomed

Reply:
They are not needed for building the binary. The binary just reads them from the container